### ISSUE TITLE
Move lesson to Carpentries Incubator

### ISSUE THREAD

**Issue by svenvanderburg** [carpentries-incubator/efficient-computing-in-julia#10]
Ideally we should move this lesson the carpentries incubator, it will help make the lesson more visible and have more impact.

You can submit it by opening an issue [through this link](https://github.com/carpentries-incubator/proposals/issues/new?assignees=tobyhodges&labels=proposal&projects=&template=lesson_proposal.yml&title=%5BProposal%5D%3A+)

**Comment by svenvanderburg** [comment]
I was at CarpentryConnect and was confronted with the fact that many people are working on the same lesson material without knowing about each others efforts.

So please [open a proposal issue](https://github.com/carpentries-incubator/proposals/issues) in 5 minutes. I could also do it for you if you want. Just to remind you that the incubator is ment to **incubate** lessons, your lesson does not need to be finished in any way and even lesson ideas are good to add to the incubator.

You can also post a message in the carpentries slack to mention that you are working on this lesson. Which I can also do for you if you want.

@jhidding

**Comment by svenvanderburg** [comment]
@jhidding

**Comment by jhidding** [comment]
Yes, sure! Sorry for not replying earlier.

**Comment by jhidding** [comment]
Note that the most up-to-date repository is now in `efficient-computing-in-julia-2`. I will transfer that here but the current repo doesn't build and I have no idea why. Sandpaper is being obnoxious again and again.

**Comment by svenvanderburg** [comment]
OK, can you archive this repo?

**Comment by jhidding** [comment]
No, there is valuable discussions in the issues concerning other online materials. I want to migrate everything here again once I have a full working version

**Comment by jhidding** [comment]
Yes

**Comment by jhidding** [comment]
Sorry for the mess, things are moving rapidly now, with the workshop coming up

**Comment by svenvanderburg** [comment]
Move fast and break things is my life motto!

**Comment by svenvanderburg** [comment]
https://github.com/carpentries-incubator/proposals/issues/205 is the proposal to migrate to carpentries incubator

**Comment by jhidding** [comment]
small typo: We **k**now about the Novice Julia lesson...

**Comment by svenvanderburg** [comment]
> small typo: We **k**now about the Novice Julia lesson...

Thanks! Fixed it 🙏

**Comment by jhidding** [comment]
The lesson is joined again and the site is rendering now. :partying_face:

**Comment by svenvanderburg** [comment]
Great :)
